src/did/did.ts

```diff
@@ -296,6 +297,24 @@ export class Did extends Logger {
     await this.setDidDocument(did, { ...(await this.getDidDocument(did)), service });
   }
 
+  /**
+   * Unlinks the current DID document from the DID
+   * @param did DID to unlink the DID document from
+   */
+  public async removeDidDocument(did: string): Promise<void> {
```

Comment:
documentation missing in docs

Reply:
Will be added with [CORE-947](https://evannetwork.atlassian.net/secure/RapidBoard.jspa?rapidView=1&selectedIssue=CORE-947). (Because method is [renamed](https://github.com/evannetwork/api-blockchain-core/blob/feature/CORE-947-discern-deactivated-DIDs/src/did/did.ts#L334) there anyway)